landscaper/pkg/controlplane/apis/imports/types_imports.go

```diff
@@ -0,0 +1,162 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package imports
+
+import (
+	"encoding/json"
+
+	landscaperv1alpha1 "github.com/gardener/landscaper/apis/core/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Please read!
+// Prerequisites:
+//  - existing etcd cluster that is accessible from the runtime cluster
+//  - there must be a Kubernetes API server that is properly set up for API aggregation
+//  - The kubeconfig for this API server has to be configured either in the field RuntimeCluster or
+//    in the field VirtualGarden in the import configuration
+// This component extends the given Kubernetes API server with the Gardener API server that serves
+// all of the Gardener resource groups (e.g for Shoots)
+// To setup API server aggregation, the Kubernetes API server has to be properly configured with configuration flags.
+// For more information how to configure the aggregation layer see:
+// https://kubernetes.io/docs/tasks/extend-kubernetes/configure-aggregation-layer
```

Comment:
Hm, why is this here? To me it sounds to better fit into a document and not into a comment inside the code?

Reply:
I will also add a more detailed version to the documentation for the component itself. 
I think that is helpful information to understand the configuration options. I would like to keep it.

Reply:
I added a dedicated README file with more detailed information and removed this comment.